posts/innovative-snowflake-features-architecture.md

```diff
@@ -0,0 +1,126 @@
+| -------------- | ------------------- | ---------------- | -----
+| X-Small        | 1                   | 1                | Default size for warehouses using CREATE
+| Small          | 2                   | 2                |
+| Medium         | 4                   | 4                |
+| Large          | 8                   | 8                |
+| X-Large        | 16                  | 16               | Default size for warehouses created in the UI
+| 2X-Large       | 32                  | 32               |
+| 3X-Large       | 64                  | 64               |
+| 4X-Large       | 128                 | 128              |
+
+*It is possible for users to script warehouses of size up to 5XL. 4XL is the maximum size when creating warehouses from the Web UI. If a warehouse of size greater than 5XL is required, you will need to work with the Snowflake Sales Team.*
+
+Warehouses can be resized at any time with no downtime or performance degradation. Any queries already running against the warehouse will complete using the current resource levels. Any queued and new queries will use the newly provisioned resource levels.
+
+By default, Snowflake will automatically suspend an unused warehouse after a period of time. Similarly, Snowflake will automatically resume a warehouse when a query using it is submitted and the warehouse is the current warehouse for the session.
+
+#### Multi-Cluster Warehouses
+Multi-Cluster Warehouses allow for the scalability of compute clusters to manage user and query concurrency needs. Typically, a virtual warehouse contains a single cluster of servers. By setting the minimum and maximum number of server clusters, Snowflake will automatically scale warehouses horizontally according to demand.
+
+*These cost more than standard Virtual Warehouses and are only available for Enterprise Edition customers and above*
+
+#### Credit Usage and Billing
+All costs for compute resources are based on Snowflake Credits. Credits are charged based on the number of Virtual Warehouses used, how long they run and their size. There is a one-to-one relationship between the number of servers in a warehouse and the number of credits they consume per hour. Warehouses are only billed when they are running. Credits are billed per-second, with a 60-second minimum. After 1 minute, all subsequent billing is per-second. When a warehouse is resized, credits are billed only for the additional servers.
+
+## 3. Global Services
+The Global Services layer coordinates and manages the entire Snowflake system. It authenticates users, manages sessions and secures data. In addition, the Global Services layer performs query optimization and compilation, as well as managing Virtual Warehouses. Using the Global Services layer, Snowflake can ensure that once a transaction on a virtual warehouse is complete, all virtual warehouses see the new data.
+
+All communication to Snowflake is encrypted from end-to-end.
```

Comment:
I'm not sure if this statement fits in the flow of the architecture and it makes me a little nervous as apart from the one line in the slides (which links to encryption of data) i cannot find it in their documentation

Reply:
Line 115 right? I was a little worried about it too. Especially since it does not fit into the actual Global Services section

Reply:
I went ahead and removed this line in commit https://github.com/ippontech/blog-usa/pull/129/commits/d892601ae4ef85c1522177215f8ec375d6a488e3